server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/SecurityConfig.java

```diff
@@ -82,20 +87,35 @@
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
 
         http
-        .csrf(csrf -> csrf.disable())
-        .authorizeHttpRequests(auth -> auth
-            .requestMatchers("/actuator/health").permitAll()
-            .anyRequest().authenticated()
-        )
-        .oauth2ResourceServer(oauth2 -> oauth2
-            .authenticationManagerResolver(authenticationManagerResolver())
-        )
-        .httpBasic()
-        .and()
-        .exceptionHandling(exceptionHandling -> exceptionHandling
-            .authenticationEntryPoint(authenticationEntryPoint()));
-
-    return http.build();
+                .csrf(csrf -> csrf.disable())
```

Comment:
API will be internal only, but still, we shouldn't need to disable csrf anymore as our only client will likely be browser based.

So, it would likely be a reasonable change to remove csrf.disable as suggested here